js.js

```diff
@@ -0,0 +1,8 @@
+function Add(){
+    
+    var wynik;
+
+    var task = document.getElementById('name').value;
```

Comment:
Znasz różnicę pomiędzy var / let / const?

Reply:
jak używasz const to jest stała i nie możesz jej póżniej zmienić a let i var to nie wiem czym sie różni